Frontend/ui-common/src/Engineering/Helpers/JBridge.purs

```diff
@@ -77,7 +77,7 @@ import Foreign.Class (encode)
 foreign import showLoaderImpl      :: String -> Effect Unit
 -- foreign import readFile'      :: String -> Effect String
 -- foreign import showLoader'      :: String -> Effect Unit
-foreign import locateOnMap :: EffectFn5 Boolean Number Number String (Array Location) Unit
+foreign import locateOnMap :: LocateOnMapConfig -> Effect Unit
```

Comment:
@dhruvSHA256 why is it changed back to curried function?

Reply:
@Vignesh-772 asked to pass params as an object instead of separate args, would be easier to add more params in future